src/scaffold.ts

```diff
@@ -31,31 +32,61 @@ type MessageResponse<MessageType extends string> = Awaited<
   ReturnType<Extract<MessageSender, (type: MessageType, body: any) => any>>
 >;
 
-const scaffoldWebviewCache = new WebviewPanelCache();
+interface PrefilledTemplateOptions {
+  templateName?: string;
+  isFormNeeded?: string | undefined;
```

Comment:
We shouldn't need this as part of the interface since it's just used in `handleProjectScaffoldUri()` but not for any of the form-filling info, right?